composer.json

```diff
@@ -9,12 +9,13 @@
             "homepage": "https://github.com/symfony-cmf/symfony-cmf/contributors"
         }
     ],
-    "minimum-stability": "dev",
+    "minimum-stability": "RC",
```

Comment:
`"jackalope/jackalope-fs": "dev-master",` no longer resolves with this stability requirement. can we switch to jackalope-doctrine-dbal (maybe with sqlite if that helps?)